src/Command/AccountExpirationNotifyCommand.php

```diff
@@ -1,20 +1,32 @@
 <?php
```

Comment:
What I'm finding odd in running this is that only 1 email is going out in testdata that has 1 expired and 2 about to expire. Only the expired email went out.